src/containers/my-certificates/cetrificate-item/certificate-item.js

```diff
@@ -2,64 +2,78 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import { TableCell, TableRow, Tooltip } from '@material-ui/core';
-import { useStyles } from './certificate-item.styles';
+
 import CertificateCodeCopy from '../../../images/certificates/certificateCodeCopy';
 import CertificateCodeGift from '../../../images/certificates/certificateCodeGift';
 import CertificateImages from '../../../images/certificates/CertificateImages';
 
+import { useActiveStyles, useNotActiveStyles } from './certificate-item.styles';
+
 const CertificateItem = ({ item, openModal }) => {
-  const styles = useStyles();
   const { t } = useTranslation();
+  const activeStyles = useActiveStyles();
+  const notActiveStyles = useNotActiveStyles();
+
+  const styles = item.isActivated ? activeStyles : notActiveStyles;
+
+  const date = new Date().toISOString();
+  const daysRemaining = Date.parse(item.dateEnd) - Date.parse(date);
+  const isLessThanMonth = daysRemaining < 2678400000 && daysRemaining > 0;
 
   const dateHandler = (date) => date.slice(0, 10).split('-').reverse().join('/');
 
   return (
-    <TableRow className={styles.root} data-cy='certificate-item'>
-      <TableCell data-cy='certificate-item-img'>
+    <TableRow className={isLessThanMonth ? styles.rootRed : styles.root}>
```

Comment:
Create Table component

Reply:
Created

Reply:
Body items render component with many props, so i need to pass all this props to table from certificateTable and OrderTable component and also check if it is table for certificate or for cart.